pom.xml

```diff
@@ -121,7 +145,7 @@
             <artifactId>snakeyaml</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.ning</groupId>
+            <groupId>org.asynchttpclient</groupId>
```

Comment:
seems we don't need to list the dependancy as it exists in killbill-oss-parent, what to you think?

Reply:
removed